examples/tutorials/crystallization_tendency_regression.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<p>Plot validation/test loss instead; training loss isn't good to track</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Done